### PR TITLE
add scopes for graph and office client in test and sample

### PR DESCRIPTION
**Pull request by zhichengliu12581**
carry on https://github.com/Azure/azure-sdk-for-java/pull/18322

### CODE REVIEW

File: sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-webapp/README.md [hunk 1]
```diff
@@ -19,16 +19,16 @@ Follow the guide [here](https://docs.microsoft.com/azure/active-directory/develo
...
+The sample retrieves user's group memberships using Microsoft graph API which requires the registered app to have `Directory.AccessAsUser.All`, `User.Read` permission under `Delegated Permissions` of `Microsoft Graph`, which can allow an application to access the directory as the signed-in user. Also, to display the function of calling multiple resources, this sample will acquire `ActivityFeed.Read`, `ActivityFeed.ReadDlp`, `ServiceHealth.Read` permission under `Office 365 Management APIs` resource. You need AAD admin privilege to be able to grant the permission in API ACCESS -> Required permission. You can follow the below steps:	
 
 * In the list of pages for the app, select **API permissions**	
    - Click the **Add a permission** button	
    - Ensure that the **Microsoft APIs** tab is selected	
    - In the *Commonly used Microsoft APIs* section, click on **Microsoft Graph**	
-   - In the **Delegated permissions** section, ensure that the right permissions are checked: **Directory.AccessAsUser.All**	
+   - In the **Delegated permissions** section, ensure that the right permissions are checked: **Directory.AccessAsUser.All**, **ActivityFeed.ReadDlp**, **ServiceHealth.Read**
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-webapp/README.md:28]: Mistake here.
image

File: sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-webapp/src/main/resources/application.yml [hunk 2]
```diff
@@ -5,9 +5,14 @@ azure:
         on-demand: true
         scopes: https://management.core.windows.net/user_impersonation
       graph:
-        scopes: https://graph.microsoft.com/Calendars.Read
+        scopes:
+            - https://graph.microsoft.com/Calendars.Read
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-webapp/src/main/resources/application.yml:9]: Please set according to this image:
image

File: sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/converter/AccessTokenScopesIT.java [hunk 3]
```diff
@@ -18,29 +18,33 @@
...
+            Assert.assertFalse(result.get("accessTokenScopes/office").contains("profile"));
+            Assert.assertTrue(result.get("accessTokenScopes/office").contains("https://manage.office.com/ActivityFeed.Read"));
+            Assert.assertTrue(result.get("accessTokenScopes/office").contains("https://manage.office.com/ActivityFeed.ReadDlp"));
+            Assert.assertTrue(result.get("accessTokenScopes/office").contains("https://manage.office.com/ServiceHealth.Read"));
 
-            Assert.assertTrue(result.get("api/azure").contains("profile"));
-            Assert.assertTrue(result.get("api/azure").contains("https://graph.microsoft.com/User.Read"));
+            Assert.assertTrue(result.get("accessTokenScopes/azure").contains("profile"));
+            Assert.assertTrue(result.get("accessTokenScopes/azure").contains("https://graph.microsoft.com/AccessReview.Read.All"));
+            Assert.assertTrue(result.get("accessTokenScopes/azure").contains("https://graph.microsoft.com/User.Read"));
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/converter/AccessTokenScopesIT.java:43]: So azure's scope is always same to graph's?

**Review by chenrujun — CHANGES_REQUESTED** [review]

File: sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/converter/AccessTokenScopesIT.java (hunk 3 above)
Comment (lzc-1997-abel) [reply sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/converter/AccessTokenScopesIT.java:43]: The scopes of each client of the same resource is the same

**Review by chenrujun — APPROVED** [review]
LGTM.

**Comment by chenrujun** [comment]
Hi, @lzc-1997-abel , please resolve conflicts.

**Pull request merged**